pypif_sdk/interop/datacite.py

```diff
@@ -0,0 +1,52 @@
+from pypif.obj import System, Reference, Person, Name
+from ..func.update_funcs import merge
+
+
+def parse_name_string(full_name):
```

Comment:
We have similar logic in jpif already https://github.com/CitrineInformatics/jpif/blob/master/src/main/java/io/citrine/jpif/obj/common/Name.java. You can just pass the entire full_name string wherever a Name object is expected, rather than try to parse it out like this.

Reply:
I'd like to keep emitting strict PIFs until we add expose a client for the standardize service.